api/src/main/java/cz/muni/fi/pa165/dto/MemberDTO.java

```diff
@@ -54,9 +54,9 @@ public void setEmail(String email) {
     public Date getRegistrationDate() {
         return registrationDate;
     }
-
-    public void setRegistrationDate(Date registrationDate) {
-        this.registrationDate = registrationDate;
+    
+    public void setRegistrationDate(Date date) {
+        this.registrationDate = date;
```

Comment:
Please don't make unnecessary changes like this, it just increases the number of possible conflicts with others